src/contracts/core/RewardsCoordinator.sol

```diff
@@ -395,13 +393,14 @@ contract RewardsCoordinator is
      * @param activatedAt The timestamp when the split is activated.
      */
     function _setOperatorSplit(OperatorSplit storage operatorSplit, uint16 split, uint32 activatedAt) internal {
+        require(split <= ONE_HUNDRED_IN_BIPS, "RewardsCoordinator._setOperatorSplit: split must be <= 10000 bips");
         require(
             block.timestamp > operatorSplit.activatedAt,
             "RewardsCoordinator._setOperatorSplit: earlier split not activated yet"
         );
         if (operatorSplit.activatedAt == 0) {
-            // If the operator split has not been initialized yet, set the old split to the default split.
-            operatorSplit.oldSplitBips = defaultOperatorSplitBips;
+            // If the operator split has not been initialized yet, set the old split to `type(uint16).max` as a flag.
+            operatorSplit.oldSplitBips = type(uint16).max;
```

Comment:
Decided to go with using `type(uint16).max` instead since the Bips variable will never have that value. And also allows us to proceed without using any additional storage.